accounting/docs.md

```diff
@@ -42,4 +42,16 @@ peers.
 - Note: A note containing the pubkey of the peer that opened a channel to us. 
 
 Known Omissions:
-- Remote peers may push balance to our node as part of the funding flow. This amount is not currently included in these reports. 
\ No newline at end of file
+- Remote peers may push balance to our node as part of the funding flow. This amount is not currently included in these reports. 
+
+### Channel Close 
+Channel close entries represent the on chain close of a channel. 
+
+- Amount: The amount that was paid out to us immediately on channel close. 
+- TxID: The on chain transaction ID for the channel close. 
+- Reference: The channel close transaction ID.
+- Note: A note indicating the type of channel close, and who initiated it. 
+
+Known Omissions: 
+- If our balance is encumbered behind a timelock, or in an unresolved htlc, it will not be paid out as part of this transaction and must be resolved by follow up on chain transactions. 
+- The fees paid to close channels that we initiated are not currently recorded, this is because balances are taken from the funding output rather than being supplied by the wallet.
```

Comment:
For anchors, we actually do pick up fees because we do the sweep back to ourselves to pickup the anchor. This is for legacy channels where fees are just portioned out of your balance.